frontend/appflowy_flutter/lib/shared/icon_emoji_picker/icon_picker.dart

```diff
@@ -246,33 +290,28 @@ class _Icon extends StatelessWidget {
   final PopoverMutex mutex;
   final void Function(BuildContext context, String color) onSelectedColor;
 
+  @override
+  State<_Icon> createState() => _IconState();
+}
+
+class _IconState extends State<_Icon> {
+  final PopoverController _popoverController = PopoverController();
+
   @override
   Widget build(BuildContext context) {
     return AppFlowyPopover(
       direction: PopoverDirection.bottomWithCenterAligned,
       offset: const Offset(0, 6),
-      mutex: mutex,
-      child: FlowyTooltip(
-        message: icon.displayName,
-        preferBelow: false,
-        child: FlowyButton(
-          useIntrinsicWidth: true,
-          margin: const EdgeInsets.all(8.0),
-          text: Center(
-            child: FlowySvg.string(
-              icon.content,
-              size: const Size.square(20),
-              color: context.pickerIconColor,
-              opacity: 0.7,
-            ),
-          ),
-        ),
+      mutex: widget.mutex,
```

Comment:
I'm a bit confused. Is the background you mentioned actually a hover effect?

Reply:
What I want to do is to allow clicking an icon to call onSelect with the group and icon immediately, without opening an extra popover to choose a background color (so onSelect will be called with null as the color)

Reply:
I understand now. I thought 'background' referred to the background effect of an icon. It actually means whether we should open the background color popup menu when selecting icons.

Reply:
Yeahhh haha I couldn't think of a better name. Do you have a suggestion?

Reply:
How about `enableBackgroundColorSelection`?

Reply:
Sounds good, I've updated this PR with the change